book.bib

```diff
@@ -510,3 +510,31 @@ @Article{mazany-wright_etal2021BulkleyRiver
   langid = {english},
   file = {D\:\\New_Graph\\References\\zotero\\al_newgraph_account\\storage\\42LVI3QP\\Mazany-Wright et al. - Bulkley River Watershed (Laxyip  Wedzin Kwah).pdf},
 }
+
+@report{irvine2020ParsnipRiver,
+  title = {Parsnip {{River Watershed}} – {{Fish Habitat Confirmations}}},
+  author = {Irvine, A},
+  date = {2020},
+  url = {https://newgraphenvironment.github.io/Parsnip_Fish_Passage/}
+}
+
+@report{irvine2021RestoringFish,
```

Comment:
date is 2022

![image](https://user-images.githubusercontent.com/46538103/227948831-d16681b9-f94c-49a0-98b9-39ddfd3feefa.png)